### PR TITLE
MAUI-4755 Custom view UG for Shimmer in .NET MAUI 

### PR DESCRIPTION
**Pull request by Priyadharshini-Periyasamy**
### **Description**

- I have added the Custom View UG content in .NET MAUI

### CODE REVIEW

File: MAUI/Shimmer/custom-view.md [hunk 1]
```diff
@@ -0,0 +1,735 @@
...
+            shimmer.CustomView = grid;
+            this.Content = shimmer;
+        }
+{% endhighlight %}
+
+{% endtabs %}
+
+## Shimmer view
+
+You can customize the `ShimmerView` using the `CustomView` property of `SfShimmer` and also you can change shape of the shimmer wave by using the property `ShapeType`.
```
Comment (florenceliliana) [inline MAUI/Shimmer/custom-view.md:384]: You can customize the `ShimmerView` using the `CustomView` property of `SfShimmer,` and also you can change the shape of the shimmer wave using the property the `ShapeType.`

**Review by florenceliliana — CHANGES_REQUESTED** [review]

File: MAUI/Shimmer/custom-view.md [hunk 2]
```diff
@@ -0,0 +1,735 @@
...
+title: About .NET MAUI Shimmer control | Syncfusion
+description: Learn here all about the Custom view support in Syncfusion .NET MAUI Shimmer (SfShimmer) control and more.
+platform: maui
+control: SfShimmer
+documentation: ug
+---
+
+# Custom view of .NET MAUI Shimmer (SfShimmer)
+
+The `CustomView` property of `SfShimmer` can be used to create your own shimmer view. The below code demonstrate how to customize the shimmer view using .NET MAUI framework elements by `CustomView`.
```
[the remark this answers is not in the document]
Comment (Priyadharshini-Periyasamy) [reply MAUI/Shimmer/custom-view.md]: Contents modified

File: MAUI/Shimmer/custom-view.md (hunk 1 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Shimmer/custom-view.md]: Contents modified

File: MAUI/Shimmer/custom-view.md [hunk 3]
```diff
@@ -0,0 +1,735 @@
...
+            shimmer.CustomView = grid;
+            this.Content = shimmer;
+        }
+{% endhighlight %}
+
+{% endtabs %}
+
+## Shimmer view
+
+You can customize the `ShimmerView` using the `CustomView` property of `SfShimmer`, and also you can change the shape of the shimmer wave using the property `ShapeType`.
```
Comment (Sasikumar3595) [inline MAUI/Shimmer/custom-view.md:384]: Improve the content,

Comment (Priyadharshini-Periyasamy) [reply MAUI/Shimmer/custom-view.md]: Improved the shimmer view content

**Pull request closed without merging**